AspNet-Install.ps1

```diff
@@ -0,0 +1,106 @@
+[cmdletbinding()]
```

Comment:
This script effectively does the following:

- download the newest `Build.RuntimeStore` nupkg
- extract the zip
- set 3 environment variables

This will be much easier to port to bash and maintain